AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -318,7 +319,7 @@ - (void)checkLatestRelease:(NSString *)updateToken
     if (self.sender == nil) {
       NSMutableDictionary *queryStrings = [[NSMutableDictionary alloc] init];
       NSMutableDictionary *reportingParametersForUpdatedRelease =
-          [self getReportingParametersForUpdatedRelease:updateToken currentInstalledReleaseHash:releaseHash distributionGroupId:distributionGroupId];
```

Comment:
I can see that you remove current release hash from some methods parameters. Can you tell me why?
You are calling `MSPackageHash()` instead of passing it around via method parameter?
Just curious.